tests/integration/test_dbapi_integration.py

```diff
@@ -461,6 +461,7 @@ def test_time_query_param(trino_connection):
     rows = cur.fetchall()
 
     assert rows[0][0] == params
+    assert cur.description[0][1] == "time(6)"
```

Comment:
same for `time with time zone`?

Reply:
Added similar assertion to `test_time_with_time_zone_negative_offset()` and `test_time_with_time_zone_positive_offset()`